### PR TITLE
Added prettier and removed README.md

### LINKED ISSUE #1: Use Web Chat's eslint files (add prettier too)
:D

### LINKED ISSUE #2: Delete README.md?
since it's empty

### PR DESCRIPTION
**Pull request by tdurnford**
Fixes #1 
Fixes #2

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 dist/**
 node_modules/**
+lib/**
```
Comment (corinagum) [inline .eslintignore]: add new line to end of file

**Comment by corinagum** [comment]
Do we want to add husky and lint-staged as well? @compulim, thoughts?

**Comment by tdurnford** [comment]
I'm planning to add husky and lint-staged, but I wanted to commit all of the files before adding them

File: src/index.ts [hunk 2]
```diff
@@ -11,4 +11,4 @@ window['MockBotAdapter'] = {
   createDirectLine
 };
 
-console.log(process.env.PUBLIC_URL)
\ No newline at end of file
+console.log(process.env.PUBLIC_URL);
```
Comment (corinagum) [inline src/index.ts]: add empty line?

Comment (tdurnford) [reply src/index.ts]: Oh I need to remove that

File: .eslintignore (hunk 1 above)
Comment (tdurnford) [reply .eslintignore]: Removing this file

**Pull request merged**